frontend/src/scenes/web-analytics/WebAnalyticsHealthCheck.tsx

```diff
@@ -49,7 +49,10 @@ export const WebAnalyticsHealthCheck = (): JSX.Element | null => {
         )
     }
 
-    if (productTab === ProductTab.WEB_VITALS) {
+    // For Page reports tab, use the same health check as the Analytics tab
```

Comment:
This change seems moot, we could have kept the previous code

Reply:
Agreed. Will decursorfy it.